ext/mbstring/config.m4

```diff
@@ -199,8 +199,6 @@ PHP_ARG_ENABLE([mbregex],
   [no])
 
 if test "$PHP_MBSTRING" != "no"; then
-  AC_DEFINE([HAVE_MBSTRING],1,[whether to have multibyte string support])
```

Comment:
It's okay to drop the checks, but I believe this define should stay, as other extensions may use it to check whether mbstring is compiled in.

Reply:
Good point.